doc/userguide/src/CreatingTestData/Variables.rst

```diff
@@ -1007,6 +1007,13 @@ can be changed dynamically using keywords from the `BuiltIn`_ library.
    +------------------------+-------------------------------------------------------+------------+
    | ${PREV TEST MESSAGE}   | The possible error message of the previous test case. | Everywhere |
    +------------------------+-------------------------------------------------------+------------+
+   | ${SETTINGS}            | A dictionary exposing some command line options used  | Everywhere |
+   |                        | for running the current test:                         |            |
+   |                        | - EXCLUDE_TAGS                                        |            |
+   |                        |   A list of tags provided by the -exclude (-e) option |            |
+   |                        | - INCLUDE_TAGS                                        |            |
+   |                        |   A list of tags provided by the -include (-i) option |            |
+   +------------------------+-------------------------------------------------------+------------+
```

Comment:
- `--exclude` and `--include` have two hyphens.
- Should use format `:option:`--exclude` to get consistent styles
- I don't think short options need to be mentioned.
- Needs `New in RF 5.0.` note at the ends.